### PR TITLE
qemu.stop_continue: raise root-cause if test is failed

### PR DESCRIPTION
**Pull request by xutian**
Excpetion raise in finally block overwrite exception occur
in key steps. Make it raise root-cause if test is failed.

ID: 1426510

Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

File: qemu/tests/stop_continue.py [hunk 1]
```diff
@@ -86,11 +86,17 @@ def run(test, params, env):
...
+                # session close if exception raised, so get renew a session
+                # to do cleanup step.
+                session = vm.wait_for_login(timeout=login_timeout)
+                op_timeout = float(params.get("clean_op_timeout", 60))
+                session.cmd(clean_op, timeout=op_timeout, ignore_all_errors=True)
+            session.close()
+            if session_bg:
+                session_bg.close()
+        except Exception, details:
+            logging.warn("Excpetion occur in the end of test: %s" % details)
```
Comment (suqinhuang) [inline qemu/tests/stop_continue.py]: typo, should be "Exception"

Comment (suqinhuang) [inline qemu/tests/stop_continue.py]: and meybe "Exception occur while cleanning env" is more better, which tell user it's the operation to clean the env

**Comment by xutian** [comment]
@CongLi  thanks your comments, and commit updated, please help to review. thanks

**Comment by xutian** [comment]
@suqinhuang  updated, can you review it again.

**Comment by xutian** [comment]
Merged it since there no such kind of issue happen again after applied this patch.

**Pull request merged**